packages/scandipwa/src/component/MyAccountOrderPopup/MyAccountOrderPopup.container.js

```diff
@@ -32,7 +32,7 @@ export const OrderDispatcher = import(
 export const mapStateToProps = (state) => ({
     order: state.OrderReducer.order,
     payload: state.PopupReducer.popupPayload[ORDER_POPUP_ID] || {},
-    display_tax_in_shipping_amount: state.ConfigReducer.cartDisplayConfig.display_tax_in_shipping_amount
+    display_tax_in_shipping_amount: state.ConfigReducer.cartDisplayConfig.display_tax_in_shipping_amount || ''
```

Comment:
I think it's better to have this value set as default value in the corresponding reducer's `getInitialState`.
Please provide this change and then we'll merge!

Reply:
Done.